pandas/tests/frame/methods/test_drop.py

```diff
@@ -268,6 +269,18 @@ def test_drop_non_empty_list(self, index, drop_labels):
         with pytest.raises(KeyError, match="not found in axis"):
             DataFrame(index=index).drop(drop_labels)
 
+    @pytest.mark.parametrize("empty_listlike", [[], DatetimeIndex([])])
```

Comment:
Did you try empty index and empty numpy array? Just curious, otherwise lgtm

Reply:
Looks like Pandas has a function for identify list-like objects (https://pandas.pydata.org/docs/reference/api/pandas.api.types.is_list_like.html) and this includes "Python lists, tuples, sets, NumPy arrays, and Pandas Series."

Just added empty Index, np.Array and pd.Series and the Test Case passes.

Reply:
Great:)